open-machine-learning-jupyter-book/ml-fundamentals/regression/Univariate-linear-regression.ipynb

```diff
@@ -0,0 +1,338 @@
+{
```

Comment:
<p>Asset needs to be uploaded to COS, use the COS link here, see the group announcement for the COS account.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/747/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>